src/locales/en.json

```diff
@@ -49,12 +49,12 @@
             "TitleUnderVoltageDetected": "Under-Voltage Detected"
         },
         "Titles": {
-            "Complete": "Complete - {filename}",
+            "Complete": "Complete - {filename} - {name}",
```

Comment:
i would recommend to change the name of these translations, that all translators see a new key to translate

Reply:
Can you give an example of what you mean? Do you want me to go through every translation and update? Or change the key name? Won't that break the other translations if I do the second one?

Reply:
That's what changing the key is precisely about. Just as an example:

You only change the English translation here. The key remains the same, so no translator is made aware that there are new translations that are not translated. If we take DE as an example, without this change, the old text is still displayed without the name.

So it is better to change the key, which does not exist as a translation, but it has a correct fallback in English with all variables or texts it should have.

An working "new" key would be "CompleteWithPrintername"...

Reply:
Ahh, understood! That makes sense - I went ahead and updated this! Thanks for clarifying!